lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -665,6 +669,11 @@ def createPanel(self, widget, name, floating=False, registerconfig=True, custom=
             self.info('Panel with name "%s" already exists. Reusing.' % name)
             return self.__panels[name]
 
+        if isinstance(widget, TaurusBaseComponent):
+            widget.setModifiableByUser(modifiablebyuser)
```

Comment:
Even if we disregard my previous comment about these options not belonging to `createPanel`, note that this implementation would break things in the case of widgets that are implemented with customized values of the affected properties (e.g., the `TaurusTrend2DDialog` is implemented with `modifiableByUser=True` by default, but with this implementation it would be set to False). 
IMHO, `createPanel()` should respect the widget's defaults at least unless explicitly overriden.

Reply:
Indeed, default values should be respected if no overriding values are provided in PanelDescription.